Description: disable failing test
Author: Marc Deslauriers <marc.deslauriers@canonical.com>
Forwarded: no

Index: b/tests/Makefile.am
===================================================================
--- a/tests/Makefile.am
+++ b/tests/Makefile.am
@@ -90,7 +90,7 @@ ctests = mini-record-2 simple gc set_pkc
 	 x509sign-verify2 mini-alignment oids atfork prf \
 	 status-request status-request-ok fallback-scsv pkcs8-key-decode \
 	 mini-session-verify-function auto-verify mini-x509-default-prio \
-	 global-init-override pcert-list
+	 pcert-list
 
 mini_dtls_pthread_LDADD = $(LDADD) -lpthread
 
